app/livechat/client/views/app/livechatDepartments.js

```diff
@@ -98,17 +97,15 @@ Template.livechatDepartments.onCreated(function() {
 
 	this.autorun(async function() {
 		const limit = instance.limit.get();
-		const { departments } = await APIClient.v1.get(`livechat/department?count=${ limit }`);
+		const filter = instance.filter.get();
+		let baseUrl = `livechat/department?count=${ limit }`;
+
+		if (filter) {
+			baseUrl = baseUrl.concat(`&text=${ encodeURIComponent(filter) }`);
```

Comment:
Please use assign operators to concat (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance)
```suggestion
			baseUrl += `&text=${ encodeURIComponent(filter) }`;
```